packages/round-manager/craco.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
           https: require.resolve('https-browserify'),
           os: require.resolve('os-browserify'),
           url: require.resolve('url'),
+          util: require.resolve('util/')
```

Comment:
should this be `util: require.resolve('util')`  ?

Reply:
fixed in #41

Reply:
done